disco_aws_automation/resource_helper.py

```diff
@@ -87,8 +84,7 @@ def throttled_call(fun, *args, **kwargs):
     (up to MAX_POLL_INTERVAL seconds).
     """
     max_time = 5 * 60
```

Comment:
With introduction of backoff() function MAX_POLL_INTERVAL became irrelevant as a global variable. I removed it from the module body and moved it to backoff() function as loca variable.